src/main/resources/htmlpublisher/workflow/PublishHTMLStep/config.jelly

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License
+
+Copyright 2015 CloudBees Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry field="target" title="${%Report}">
+    <table width="100%">
+      <col width="20%"/>
+      <col width="20%"/>
+      <col width="20%"/>
+      <col width="20%"/>
+      <col width="20%"/>
+      <tr>
+        <td>HTML directory to archive</td>
+        <td>Index page[s]</td>
+        <td>Report title</td>
+        <td>Keep past HTML reports</td>
+        <td>Allow missing report</td>
+        <td/>
+      </tr>
+    </table>
+
+    <table width="100%">
+      <col width="20%"/>
+      <col width="20%"/>
+      <col width="20%"/>
+      <col width="20%"/>
+      <col width="20%"/>
+      <tr>
+        <td>
+          <f:textbox field="reportDir" />
+        </td>
+      
+        <td>
+          <f:textbox field="reportFiles"  default="index.html"/>
+        </td>
+        
+        <td>
+          <f:textbox field="reportName" default="HTML Report"/>
+        </td>
+        
+        <td>
+          <f:checkbox field="keepAll" />
+        </td>
+
+        <td>
+          <f:checkbox field="allowMissing" />
+        </td>
+      </tr>
+    </table>
```

Comment:
Simpler to kill the `table`, which is not useful in the context in which step’s configuration would be displayed anyway.